python/ray/experimental/state/api.py

```diff
@@ -174,7 +174,7 @@ def _make_param(cls, options: Union[ListApiOptions, GetApiOptions]) -> Dict:
                 continue
 
             option_val = getattr(options, field.name)
-            if option_val:
+            if option_val is not None:
```

Comment:
this is a bug....

Reply:
what's the bug?

Reply:
If the `option_val` is falsy, like False, or 0, it will be ignored.

Reply:
that makes sense.. 😞